appveyor.yml

```diff
@@ -33,9 +33,8 @@ build_script:
     - cd "%APPVEYOR_BUILD_FOLDER%"\scintilla\win32
     - if "%configuration%"=="Unicode Debug" set scintilla_debug=DEBUG=1
     - if "%configuration%"=="Unicode Release" set scintilla_debug=
-    - if "%archi%" NEQ "" nmake SUPPORT_XP=1 %scintilla_debug% -f scintilla.mak
+    - if "%archi%" NEQ "" nmake SUPPORT_XP=1 %scintilla_debug% BOOSTPATH=C:\Libraries\boost_1_69_0\ BOOSTREGEXLIBPATH=C:\Libraries\boost_1_69_0\lib32-msvc-14.1\ -f scintilla.mak
```

Comment:
I guess you need to download the boost first. Maybe, use a package from NuGet or some other package manager?
Also, it could be checked out from git repo and built. However, that might slowdown AppVeyor build significantly.

Reply:
No, I don't. Everything is pre-installed. I'm trying to check out where exactly right now.

Reply:
It will not slow down the Appveyor build.

 What I found out yet:
- The Appveyor doc is correct. For MS2017, only boost 1.69 is available
- Boost 1.69 works for the 32-bit version.

Next:
- Make it work for the 64-bit version.

Reply:
Test 3 revealed, that Boost 1.70 is not available for MS VC 2017.

Reply:
`SUPPORT_XP=1` is not necessary anymore.